src/lowertest/README.md

```diff
@@ -118,16 +138,28 @@ enums and structs are keyed by string.
 ### 3. Feeding in type information
 
 Enum/struct information must be passed into the converter via
-the struct `ReflectedTypeInfo`. For your convenience, there is a macro to
-generate a function will produce the right `ReflectedTypeInfo` for the object
-you are trying to create.
+the struct `ReflectedTypeInfo`.
+
+To populate the `ReflectedTypeInfo` struct, add these two lines to your code:
 ```
-get_reflect_info_func!(func_name, [EnumType1, .., EnumTypeN], [StructType1, ..,StructTypeN])
+let mut rti = ReflectedTypeInfo::default();
+<Object_type>::add_to_reflected_type_info(&mut rti);
 ```
-Calling `func_name()` will produce the desired `ReflectedTypeInfo`.
 
-For instructions on how to manually populate `ReflectedTypeInfo`, see the struct
-documentation.
+Provided your object type is a [supported type](#supported-types), the
+`add_to_reflected_type_info` function will recursively add types your object
+type depends on to `ReflectedTypoInfo`.
+
+Notably, the type `Result<<ok_type>, <error_type>>` is not yet
+supported, so you may find that you need to separately add information about the
+error type to `ReflectedTypoInfo`.
+```
+<error_type>::add_to_reflected_type_info(&mut rti)
+```
+
+If a field has the attribute `#[mzreflect(ignore)]`, then
+`add_to_reflected_type_info` will not add the type of the field, but it will add
+the types of the other fields of the enum variant or struct.
 
 ### 4. Feeding in syntax extensions
```

Comment:
It would be really helpful here to include some example from the code with a brief explanation of what it does/why it was necessary. It looks like `MirScalarExprDeserializeContext` and `MirRelationExprDeserializeContext` do this, so correlating `MirScalarExpr` and `MirRelationExpr` with these in the doc would help someone figure this out in the future.

Reply:
I have not added a code example, but I hope this rewritten paragraph is clearer.